chadrick-d-dev/hang-in-there-boilerplate
# Finished Iteration 2 functions so

### What's this PR do
Completed unfinished function from last pull request. Added an event listener for the showMyPoster button, that included creating a new poster from user input, and displaying that poster on the main view.

Also updated the README.md for iteration 2.

Removed a console.log from the main.js

### Where should the reviewer start?
line 126 of main.js

line 33 of README.md

### How should this be manually tested?
open index.html
click Make Your Own Poster button
fill in the 3 input fields on the form view
click Show My Poster button 
in Dev tools, console.log(images, titles, quotes); to see created poster
